post.hbs

```diff
@@ -31,6 +21,19 @@
                             </span>
                         {{/foreach}}
                     </span>
+                    <span>
+                        {{reading_time}}
+                    </span>
+                </span>
+
+                <span class="flex flex-wrap content-center gap-4 justify-center"> 
+                {{#foreach tags}}
+                    <div class="flex-none text-base text-black hover:underline border-2 border-black hover:border-yellow-500 rounded-full px-3">
+                        <a href={{url}}>
```

Comment:
On second thought, I think having underline for this doesn't fit well... could you add a class to disable the underline on hover?

Reply:
Also, can the text here be of the same colour as the `span` above?

Reply:
"same colour as the span above?"

Im not sure which element youre referring to. It could be:

1. Light grey "Written By:" color
2. Black Title color (no difference)

Reply:
I'm referring to the light grey color. Apologies for the ambiguity!

Reply:
Done